uber/configspec.ini

```diff
@@ -189,6 +189,9 @@ url_base = string(default="%(url_root)s%(path)s")
 default_url = string(default="%(path)s")
 default_url_priority = integer(default=1)
 
+# This determines how many rows we'll load at once for the homepage
+row_load_limit = integer(default=2000)
```

Comment:
This seems like a high default!  We'll definitely need to do some performance testing between now and Super.